index.html

```diff
@@ -202,6 +202,12 @@ <h1>The Registration Process</h1>
 description of the addition.
       </li>
       <li>
+The name of the extension MUST be indicative of its function, or - in the case
```

Comment:
Procedural question: Can we put normative advice in this document or does it need to be put in did core?

Reply:
My understanding is that the registry document itself defines the requirements for listing in the registry. The key is that a WG needs to formally approve those rules. Such changes are never "editorial". So, I believe its fair game, but we should vet the rule changes with a proposal in the regular WG channels.

Reply:
Sounds good, I agree with that approach. Thanks for weighing in on it.